games-fps/alephone/alephone-20180614.recipe

```diff
@@ -0,0 +1,137 @@
+SUMMARY="Open source continuation of Bungie’s Marathon 2 game engine"
+DESCRIPTION="Aleph One plays Marathon, Marathon 2, Marathon Infinity, and \
+3rd-party content on a wide array of platforms, with (optional) OpenGL \
+rendering, Internet play, Lua scripting, and more."
+HOMEPAGE="https://alephone.lhowon.org/"
+COPYRIGHT="1991-2001 Bungie Software
+	Aleph One Developers"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="b5ac7ae645404e1f23d2dca388373195bcac5504"
+SOURCE_URI="https://github.com/Aleph-One-Marathon/alephone/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="711eddff5c139cf523ddda4f2f9c6da30a7e52feb668bef8dec041614747f189"
+SOURCE_DIR="alephone-$srcGitRev"
+PATCHES="alephone-$portVersion.patchset"
+ADDITIONAL_FILES="alephone.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+    commandSuffix=
+    commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	alephone$secondaryArchSuffix = $portVersion
+	cmd:alephone$commandSuffix = $portVersion
```

Comment:
Should we instead install this as an app?

Reply:
I'd say :
* **_yes_** for :
  * `alephone_marathon-20150907.recipe`
  * `alephone_marathon2-20150907.recipe`
  * `alephone_marathoninfinity-20150907.recipe`

* and **_no_** for :
  * `alephone-20180614.recipe`

(That's because the last recipe is just an engine.)
That said, I'm OK with any other point of view.


BTW, I've just noticed that we need to have separate directories for each recipe. That's because the build directories of 3 of the 4 recipes would all have the same name, `work-20140907`, so we would not be able to build all three recipes in a single call to haikuporter because we would have to clearn the work directory before each build.

So I'd recommend to move :
* `alephone_marathon-20150907.recipe` to `games-fps/alephone-marathon/`
* `alephone_marathon2-20150907.recipe` to `games-fps/alephone-marathon2/`
* `alephone_marathoninfinity-20150907.recipe` to `games-fps/alephone-marathoninfinity/`

(Of course, you can use underscores in the names of the directories, but I think it would be nicer without the underscores.)